backend/app/main.py

```
@@ -23,8 +23,27 @@
     paybill,
 )
 
-import sentry_sdk
 from fastapi.middleware.cors import CORSMiddleware
+import uvicorn
```

Ruff (0.8.2)

Redefinition of unused `uvicorn` from line 1

Remove definition: `uvicorn`

(F811)